### PR TITLE
Line wrapping and spacing

### PR DESCRIPTION
**Pull request by chrisfinazzo**

### CODE REVIEW

**Comment by mattr-** [comment]
Depending on the markdown processor involved and it's default options, you get different results in the rendering of said markdown when changing the line breaks like this. Was there a particular issue you were trying to solve with the rendering of the site with this PR?

**Comment by envygeeks** [comment]
@mattr- it's the same thing he's been doing for weeks, that was one of my initial concerns but it seems most markdown processors are consistent on this subject now, they just make <br> hard and manual and allow pr line breaks without concern.

**Comment by envygeeks** [comment]
I say that because I tested it with RedCarpet, Kramdown, Github-Markdown and PHP Markdown

**Comment by chrisfinazzo** [comment]
@mattr, 

To your point, it's mostly for consistency of style and to fix a few typos that I noticed hadn't been caught. For simplicity, I kept Kramdown as the processor in that branch, although I see @envygeeks did some testing with the others...

As for the build errors, Travis chokes on Sass and Pygments.rb (gems are corrupt, according to their log) although `bundle exec rake test` and `script/cibuild` ran with no errors locally. I did notice that my job sat in the queue for what seemed like a longer time than usual, but that might just be due to the maintenance described [here][] and things coming back online.

[here]: http://www.traviscistatus.com/incidents/h91j8ft0qs2l

File: site/_docs/templates.md [hunk 1]
```diff
@@ -306,14 +307,15 @@ You can also choose to include file fragments relative to the current file:
 {% raw %}{% include_relative somedir/footer.html %}{% endraw %}
 {% endhighlight %}
 
-You won't need to place your included content within the `_includes` directory. Instead,
-the inclusion is specifically relative to the file where the tag is being used. For example,
-if `_posts/2014-09-03-my-file.markdown` uses the `include_relative` tag, the included file
-must be within the `_posts` directory, or one of its subdirectories. You cannot include
-files in other locations.
+You won't need to place your included content within the `_includes`
+directory.Instead, the inclusion is specifically relative to the file where the
```
Comment (parkr) [inline site/_docs/templates.md]: Space was omitted in this line.

File: site/_docs/templates.md [hunk 2]
```diff
@@ -306,14 +307,15 @@ You can also choose to include file fragments relative to the current file:
...
+tag is being used. For example,if `_posts/2014-09-03-my-file.markdown` uses the
```
Comment (parkr) [inline site/_docs/templates.md]: again here.

**Comment by parkr** [comment]
I'm happy to take the typo fixes but I don't think the line wrapping fixes are worth messing with the git history.

**Comment by envygeeks** [comment]
We aren't altering the history though?

**Comment by parkr** [comment]
> We aren't altering the history though?

Rewriting the lines will add more hoops to jump through. Maybe my git fu isn't up to par, but historically it has made it more difficult to find where a particular piece of the documentation was altered.

**Comment by envygeeks** [comment]
It does do that, I definitely agree with that!

**Comment by chrisfinazzo** [comment]
Going to close this out and just do the typo corrections inside of `troubleshooting.md`. See #3683.

**Pull request closed without merging**